src/workerd/server/workerd-api.c++

```diff
@@ -1064,13 +1041,14 @@ kj::Own<jsg::modules::ModuleRegistry> WorkerdApi::initializeBundleModuleRegistry
 
   builder.add(bundleBuilder.finish());
 
+  // Add the built-in module bundles that support python workers/pyodide.
   if (hasPythonModules) {
     jsg::modules::ModuleBundle::BuiltinBuilder pyodideBundleBuilder;
-    auto metadataSpecifier = "pyodide-internal:runtime-generated/metadata"_url;
-    auto artifactsSpecifier = "pyodide-internal:artifacts"_url;
-    auto internalJaegerSpecifier = "pyodide-internal:internalJaeger"_url;
-    auto diskCacheSpecifier = "pyodide-internal:disk_cache"_url;
-    auto limiterSpecifier = "pyodide-internal:limiter"_url;
+    const auto metadataSpecifier = "pyodide-internal:runtime-generated/metadata"_url;
```

Comment:
Can these be constexpr?

Reply:
Not currently. The `_url` operator does not support it at this time. We can do that separately.